db_release_scripts/README.MD

```diff
@@ -1,144 +1,145 @@
 
 * **mosip_<schema_name>/<schema_name>_revoke_db_deploy.sh:**  Script to revoke database deployment.
 
-**Note:** SQL script files name under sql folder are appended with release version numbers( like **1.2.0-SNAPSHOT**_master-scripts_release.sql), Same version number has to be passed as argument while executing release/revoke shell scripts. Ex. **$bash master_release_db_deploy.sh master_release_deploy.properties 1.2.0-SNAPSHOT**
 
-**Note :** No need of shell script update unless it is really causing any problem or any further implementation is being introduced.
+### Log File Directory
+
+Create a log file directory if it doesn't exist on DB deployment server. Follow the steps:
+ 
+``` bash-4.2$mkdir /mosip_base_directory/<folder_name> ```
+
+Log files can be placed in a different folder by specifying the path of the directory in the properties file.
+
+**Note:** SQL script files name under sql folder are appended with release version numbers( like **1.2.0-SNAPSHOT**_kernel-scripts_release.sql), that version number should be passed as argument while executing release/revoke shell scripts. Ex. **$bash master_release_db_deploy.sh master_release_deploy.properties 1.2.0-SNAPSHOT**.
 
-Once completed with sourcing the database release files, Follow the DB release deployment steps by modifying the properties file according the requirement.
+**Note :** No need to update shell script unless any error or any further implementation is being introduced.
 
 
+### Modifying Properties.
 
-### Properties file variable details and description. Properties file has to be updated with the required details  for each database before proceeding with deployment steps.
+Properties file variable details:
 
-**DB_SERVERIP=** Contains details of Destination SERVER_IP(Ex:10.0.0.1) where the deployment is targeted
+**DB_SERVERIP=** Contains details of Destination SERVER_IP(Ex:10.0.0.1) where the deployment is targeted.
 
-**DB_PORT=** Contains the postgres server port details where the postgres is allowed to connect. Ex: 5433
+**DB_PORT=** Contains the postgres server port details where the postgres is allowed to connect. Ex: 5433.
 
-**SU_USER=** Contains the postgres super user name to connect to the postgres database i.e. postgres
+**SU_USER=** Contains the postgres super user name to connect to the postgres database i.e. postgres.
 
-**DEFAULT_DB_NAME=** Default database name to connect with respective postgres server i.e. ex: postgres
+**DEFAULT_DB_NAME=** Default database name to connect with respective postgres server i.e. ex: postgres.
 
 **MOSIP_DB_NAME=** Database name for which the deployment is scheduled.
 
-**SYSADMIN_USER=** Variable contains the sysadmin use name which indeed is going to be the super user for the remaining actions going to be performed by shell script.
+**SYSADMIN_USER=** Variable contains the sysadmin user name which is super user for the remaining actions going to be performed by shell script.
```

Comment:
Variable contains the sysadmin user name which is the super user for the remaining actions that are going to be performed by shell script.